edge/pkg/edged/edged.go

```diff
@@ -543,14 +545,17 @@ func newEdged(enable bool) (*edged, error) {
 		return nil, err
 	}
 
-	// create a log manager
-	logManager, err := logs.NewContainerLogManager(runtimeService, ed.os, "10Mi", 5)
-	if err != nil {
-		return nil, fmt.Errorf("new container log manager failed, err: %s", err.Error())
+	if ed.containerRuntimeName == remoteContainerRuntime {
```

Comment:
can be replaced by constant `RemoteContainerRuntime` in package "k8s.io/kubernetes/pkg/kubelet/types"

Reply:
/lgtm